### PR TITLE
ch-setup-react-project-#181414550

### PR DESCRIPTION
**Pull request by SergeNm**
## What does this PR do?
- Set up an empty react project

## Description of task
- The team should create a react project from scratch using Webpack 4, Babel, and the Airbnb linting option.

## How to test this PR?
- install nodejs and **yarn** package manager
- run **yarn install**
- run "npm run "**eslint-fix**" to see eslint working
- then run "**yarn run dev**"

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/app';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (rukundo-kevin) [inline src/index.js]: ReactDOM.render is no longer supported in React 18. Use createRoot instead
```suggestion
const root = ReactDOM.createRoot(document.getElementById('app'))
root.render(
    <App />
)

**Review by rukundo-kevin — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>React, Webpack, Babel Starter Pack</title>
```
Comment (rukundo-kevin) [inline src/index.html]: Change the title, This won't always be a starter pack

**Review by rukundo-kevin — CHANGES_REQUESTED** [review]

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: path.join(__dirname, 'src', 'index.js'),
```
Comment (NNesta) [inline webpack.config.js:5]: use es6

**Review by NNesta — CHANGES_REQUESTED** [review]

File: src/components/app.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const App = () => <div className="root">Barefoot</div>;
+
+export default App;
```
Comment (iyaremye-faustin) [inline src/components/app.js:5]: Change app.js to App.js (Good practice for components naming)

**Review by iyaremye-faustin — COMMENTED** [review]
Good work

**Review by rukundo-kevin — APPROVED** [review]

**Review by NNesta — APPROVED** [review]
Good job

**Pull request merged**